contentcuration/contentcuration/viewsets/assessmentitem.py

```diff
@@ -135,10 +139,8 @@ class Meta:
             "deleted",
         )
         list_serializer_class = AssessmentListSerializer
-        # Use the assessment_id as the lookup field for updates
-        # this may cause poor performance on updates as this field is not
-        # indexed. Monitor and potentially add an index.
-        update_lookup_field = "assessment_id"
+        # Use the contentnode and assessment_id as the lookup field for updates
+        update_lookup_field = ("contentnode", "assessment_id")
```

Comment:
Aside from linting looks like this is the only major change, is that right?

Reply:
Most of the other changes in this file are substantive - the linting has changed because those changes altered line length somewhat.

See above where it is explicitly registering the assessment_id field on the serializer.

Below where 'copy' is requiring that the target for the copy is a viewable assessment item by the user.

The change in the look up field then also affects the item lookups in the copy method.

Finally, the serialization of the AssessmentItem copying was failing because it was trying to return the serializer, and not the serializer data.